src/main/java/com/example/Restaurant_Management_System_REST_API/model/entity/Supplier.java

```diff
@@ -1,23 +1,39 @@
 package com.example.Restaurant_Management_System_REST_API.model.entity;
 
 import com.example.Restaurant_Management_System_REST_API.model.ContactDetails;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
 import jakarta.persistence.*;
+import jakarta.validation.Valid;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 import java.util.List;
 
 @Entity
 @Getter
 @Setter
+@NoArgsConstructor
 public class Supplier {
+
+    public Supplier(Long id, ContactDetails contactDetails, List<InventoryItem> inventoryItemList) {
+        this.id=id;
+        this.contactDetails=contactDetails;
+        this.inventoryItemList=inventoryItemList;
+    }
```

Comment:
yes, good eye! because I was playing and writing it by hand to check if I remember how to it and I forgot to replace it ;)
ok done! replaced with AllArgsConstructor